src/features/rssiSlice.ts

```diff
@@ -120,6 +120,12 @@ const rssiSlice = createSlice({
             if (state.isPaused) {
                 return;
             }
+
+            if (!state.serialPort || !state.serialPort.isOpen) {
+                state.data = initialData();
+                state.dataMax = [];
+                return;
+            }
```

Comment:
serial port is not closed immediately, not sure that is the correct fix.
also I think this check should go before ```state.isPaused``` check, just in case same can happen after pausing (didn't test such test case)

Reply:
I believe it is as the cart first gets cleared and then data would once in a blue moon show up for mill sec.

ClearRssiData is only called on close of serial port